wagtail/tests/test_sites.py

```diff
@@ -114,34 +114,38 @@ def setUp(self):
             hostname="example.com", port=80, root_page=Page.objects.get(pk=2)
         )
 
+    def test_dummy_request(self):
+        request = get_dummy_request(site=self.site)
+        self.assertEqual(Site.find_for_request(request), self.site)
+
     def test_with_host(self):
-        request = HttpRequest()
+        request = get_dummy_request()
         request.META = {"HTTP_HOST": "example.com", "SERVER_PORT": 80}
```

Comment:
If the primary motivation for using `get_dummy_request` here is to provide `find_for_request` with a more realistic set of META variables, reassigning META here is defeating that intent. Maybe this should become `request.META.update({...})` instead?